.taskcluster.yml

```diff
@@ -175,6 +175,37 @@ tasks:
           owner: ${user}@users.noreply.github.com
           source: ${repository}/raw/${head_rev}/.taskcluster.yml
 
+      - $if: 'tasks_for == "github-push" && head_branch[:10] == "refs/tags/"'
+        then:
+          taskId: {$eval: as_slugid("update_hook")}
+          dependencies:
+            - {$eval: as_slugid("docker_build")}
+          scopes:
+            - hooks:modify-hook:project-relman/bugbug
+            - assume:hook-id:project-relman/bugbug
+          created: {$fromNow: ''}
+          deadline: {$fromNow: '1 hour'}
+          provisionerId: aws-provisioner-v1
+          workerType: github-worker
+          payload:
+            features:
+              taskclusterProxy:
+                true
+            maxRunTime: 3600
+            image: mozilla/taskboot:latest
+            command:
+              - "/bin/sh"
+              - "-lcxe"
+              - "git clone ${repository} /code &&
+                 cd /code &&
+                 git checkout ${head_rev} &&
```

Comment:
It would be better to use the clone feature of taskboot.
The payload would be:
```yaml
command:
  - taskboot
  - build-hook
  - taskcluster-hook-pipeline-start.json
  - project-relman
  - bugbug
env:
  GIT_REPOSITORY: ${repository}
  GIT_REVISION: ${head_rev}
```

Reply:
I've quickly made this change before merging